lib/src/code/code.dart

```diff
@@ -437,34 +438,42 @@ class Code {
     final firstLine = lines.lines[block.firstLine + 1]; //Keep 1st line visible.
     final lastLine = lines.lines[block.lastLine];
 
+    // Exclude \n from the last line
+    var endOfRange = lastLine.textRange.end - 1;
+    if (lastLine.text[lastLine.text.length - 1] != '\n') {
+      endOfRange++;
+    }
```

Comment:
There was an issue with folding the block, who's last line didn't have '\n'.

python example:
```python
def myFunc():
  method1()
```

```python
def myFunc():)
```

Reply:
That's why I added the check for '\n' symbol, because we want to remove only it.